react-hot-button-component/hot-button-app/src/App.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import './App.css';
+
+function App() {
+  const [count, setCount] = useState(0);
+  const [color, setColor] = useState('');
+  useEffect(() => {
```

Comment:
We haven't learned `useEffect` yet. You won't need it in any of the exercises we do until we have an exercise that covers it.

As it turns out, this isn't a very good use of `useEffect`. You usually only use `useEffect` when you are interacting with non-React things, such as the network.